clio-client/src/test/scala/org/broadinstitute/clio/client/dispatch/DeliverArraysExecutorSpec.scala

```diff
@@ -1,114 +0,0 @@
-import org.broadinstitute.clio.transfer.model.arrays.{
-  ArraysExtensions,
-  ArraysKey,
-  ArraysMetadata
-}
-import org.broadinstitute.clio.util.model.Location
-import org.scalamock.scalatest.AsyncMockFactory
-
-class DeliverArraysExecutorSpec extends BaseClientSpec with AsyncMockFactory {
-  behavior of "DeliverArraysExecutor"
-
-  private val theKey = ArraysKey(
-    chipwellBarcode = Symbol("abcdefg"),
-    version = 1,
-    location = Location.GCP
-  )
-  private val metadata = ArraysMetadata(
-    vcfPath = Some(URI.create(s"gs://bucket/the-vcf${ArraysExtensions.VcfGzExtension}")),
-    vcfIndexPath =
-      Some(URI.create(s"gs://bucket/the-vcf${ArraysExtensions.VcfGzTbiExtension}")),
-    gtcPath = Some(URI.create(s"gs://bucket/the-gct${ArraysExtensions.GtcExtension}")),
-    grnIdatPath =
-      Some(URI.create(s"gs://bucket/the-grn${ArraysExtensions.IdatExtension}")),
-    redIdatPath =
-      Some(URI.create(s"gs://bucket/the-red${ArraysExtensions.IdatExtension}")),
-    refFastaPath =
-      Some(URI.create(s"gs://bucket/the-ref${ArraysExtensions.FastaExtension}")),
-    refFastaIndexPath =
-      Some(URI.create(s"gs://bucket/the-ref${ArraysExtensions.FastaFaiExtension}")),
-    refDictPath =
-      Some(URI.create(s"gs://bucket/the-ref${ArraysExtensions.DictExtension}"))
-  )
-  private val workspaceName = "the-workspace"
-  private val destination = URI.create("gs://the-destination/")
-
-  it should "add IO ops to copy the idats, and add workspace name" in {
-    val ioUtil = mock[IoUtil]
-    Seq.concat(metadata.vcfPath, metadata.vcfIndexPath, metadata.gtcPath).foreach { uri =>
-      (ioUtil.isGoogleObject _).expects(uri).returning(true)
-    }
-
-    val executor =
-      new DeliverArraysExecutor(DeliverArrays(theKey, workspaceName, destination))
-
-    executor.buildMove(metadata, ioUtil).runWith(Sink.head).map {
-      case (newMetadata, ops) =>
-        val movedGrn = metadata.grnIdatPath.map(
-          Metadata.buildFilePath(
-            _,
-            destination.resolve(DeliverArraysExecutor.IdatsDir),
-            ArraysExtensions.IdatExtension
-          )
-        )
-        val movedRed = metadata.redIdatPath.map(
-          Metadata.buildFilePath(
-            _,
-            destination.resolve(DeliverArraysExecutor.IdatsDir),
-            ArraysExtensions.IdatExtension
-          )
-        )
-
-        val grnMove = metadata.grnIdatPath.zip(movedGrn).map {
-          case (src, dest) => CopyOp(src, dest)
-        }
-        val redMove = metadata.redIdatPath.zip(movedRed).map {
-          case (src, dest) => CopyOp(src, dest)
-        }
-
-        newMetadata.workspaceName should be(Some(workspaceName))
-        newMetadata.grnIdatPath should be(movedGrn)
-        newMetadata.redIdatPath should be(movedRed)
-
-        ops should contain allElementsOf Seq.concat(grnMove, redMove)
-    }
-  }
-
-  it should "fail if no red idat is registered to a document" in {
```

Comment:
it looks like the negative tests weren't ported, was there a reason for that?

Reply:
Checking these cases got dropped from the main code